Makefile

```
@@ -64,41 +76,78 @@ help: ## Show available make targets
-		if ! $(call mamba_env_exists); then \
-			echo "Creating environment '$(MAMBA_ENV)'..."; \
-			micromamba create -y -f $(MAMBA_SPEC); \
-		elif [ "$(CI)" != "true" ]; then \
-			micromamba env update -n $(MAMBA_ENV) -f $(MAMBA_SPEC); \
-		fi; \
-		echo "Micromamba environment '$(MAMBA_ENV)' is ready."; \
-		mkdir -p $(BUILD_DIR) && touch $(ENV_STAMP); \
-	fi
-
-install: setup-env ## Create the micromamba env and install pre-commit hooks
+setup-env: ## Setup selected environment manager (ENV_MANAGER=micromamba|mise|system)
```

checkmake (0.3.2)

[warning] Target body for "setup-env" exceeds allowed length of 5 lines (57).

(maxbodylength)